src/System.Threading.Tasks.Dataflow/tests/project.json

```diff
@@ -27,7 +29,6 @@
     "netstandard1.3": {}
   },
   "supports": {
-    "coreFx.Test.netcore50": {},
```

Comment:
Yeah, it would be very difficult. I was originally thinking that we had infrastructure in our internal test system for something like this, but I must have been mistaken. You're right, we'll either need to support Process on netcore50, or have a test-only version of it, since I'm guessing the necessary API's are blocked by WACK.